HHS/OPRE-OPS
# Ops 441 cannot lock yourself

## What changed

Do not allow the `USER_ADMIN` to deactivate themselves.

## Issue

https://github.com/HHS/OPRE-OPS/issues/441

## How to test

1. Log in as the `Budget Team` user.
2. Navigate to the `User Admin` page.
3. Select the `Budget Team` user.
4. In the card for the user, attempt to select a `status` of `INACTIVE` or `LOCKED`
5. An error alert should appear.

## Definition of Done Checklist
